packages/edit-post/src/components/layout/index.js

```diff
@@ -646,7 +622,6 @@ function Layout( {
 						templateId={ templateId }
 						className={ className }
 						forceIsDirty={ hasActiveMetaboxes }
-						contentRef={ paddingAppenderRef }
```

Comment:
Wondering if contentRef is still needed or if we can remove the prop from the editor components.